test/bulk-update/reporter/reporter.test.js

```diff
@@ -0,0 +1,50 @@
+import { expect } from '@esm-bundle/chai';
+import BaseReporter from '../../../bulk-update/reporter/reporter.js';
+
+describe('BaseReporter', () => {
+  let reporter;
+
+  beforeEach(() => {
+    reporter = new BaseReporter();
+  });
+
+  it('should initialize with an empty report', () => {
+    const report = reporter.getReport();
+
+    expect(report.logs).to.be.an('object').that.is.empty;
+  });
+
+  it('should log a message and update the report', () => {
+    reporter.log('topic', 'status', 'message', 'arg1', 'arg2');
+    const report = reporter.getReport();
+
+    expect(report.logs.topic).to.have.lengthOf(1);
+    expect(report.logs.topic[0]).to.deep.equal({
+      status: 'status',
+      message: 'message',
+      0: 'arg1',
+      1: 'arg2',
+    });
+  });
+
+  it('should calculate the totals for each topic and status', () => {
```

Comment:
as mentioned above, this test may be a good stand in to remove some or modify the `calculateTotals` tests on the child objects.